.github/workflows/build-and-test.yml

```diff
@@ -515,10 +580,16 @@ jobs:
         with:
           name: binaries-linux_${{ matrix.ARCH }}
           path: ./bin
-      
+
+      - uses: actions/download-artifact@v3
+        if: ${{ matrix.ARCH == 'amd64' }}
+        with:
+          name: agent-bundle-linux-${{ matrix.ARCH }}
+          path: ./dist
+
       - name: Build ${{ matrix.ARCH }} docker image
         run: |
-          make docker-otelcol SKIP_COMPILE=true ARCH=${{ matrix.ARCH }}
+          make docker-otelcol SKIP_COMPILE=true SKIP_BUNDLE=true ARCH=${{ matrix.ARCH }}
```

Comment:
guessing `SKIP_BUNDLE=true` will be removed entirely later when we clean up.

Reply:
Since the bundle is built by the upstream `agent-bundle-linux` job, we can skip building it again here and other downstream jobs.